js_modules/dagit/packages/core/src/assets/LaunchAssetExecutionButton.tsx

```diff
@@ -294,6 +298,9 @@ export const LAUNCH_ASSET_EXECUTION_ASSET_NODE_FRAGMENT = gql`
         name
       }
     }
+    requiredResources {
+      resourceKey
+    }
```

Comment:
Thanks for putting this in here! Nice to defer fetching this info until you click the button 🙌